<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>626</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
   <item>
    <widget class="PyDMArchiverTimePlot" name="time_plot">
     <property name="toolTip">
      <string>Right-click to open plot options</string>
     </property>
     <property name="toolTipDuration">
      <number>5000</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="control_layout">
     <item>
      <layout class="QVBoxLayout" name="signal_layout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLineEdit" name="input_field">
         <property name="placeholderText">
          <string>PV:Enter:To:Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="curve_list"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="list_ctrl_layout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="redraw_button">
           <property name="text">
            <string>Redraw Curves</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear_button">
           <property name="text">
            <string>Clear Curves </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="button_day">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>1d</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_week">
         <property name="text">
          <string>1w</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_month">
         <property name="text">
          <string>1M</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMTimePlot</class>
   <extends>QGraphicsView</extends>
   <header>pydm.widgets.timeplot</header>
  </customwidget>
  <customwidget>
   <class>PyDMArchiverTimePlot</class>
   <extends>PyDMTimePlot</extends>
   <header>pydm.widgets.archiver_time_plot</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
